lib/widget/confirmation_input.dart

```diff
@@ -39,6 +39,15 @@ class ConfirmationInput extends StatefulWidget
           _controller.enableCursor = Utils.getBool(newValue, fallback: true),
       'length': (newValue) =>
           _controller.length = Utils.getInt(newValue, fallback: 4),
+      'width': (value) => _controller.fieldWidth = Utils.optionalDouble(value),
+      'height': (value) =>
+          _controller.fieldHeight = Utils.optionalDouble(value),
+      'padding': (value) =>
```

Comment:
not quite the same as fieldPadding, but use `gap`. We already have that concept.

Reply:
Okay I'll change it `gap` property